<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsViewerMainWindowClass</class>
 <widget class="QMainWindow" name="EventsViewerMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LANAnalyzer events viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/16_viewer.png</normaloff>:/images/16_viewer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>No.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Event</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Details</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
     <widget class="QMenu" name="menuStyle">
      <property name="title">
       <string>&amp;Style</string>
      </property>
      <addaction name="actionIconOnly"/>
      <addaction name="actionTextOnly"/>
      <addaction name="actionTextBesideIcon"/>
      <addaction name="actionTextUnderIcon"/>
     </widget>
     <widget class="QMenu" name="menuSize">
      <property name="title">
       <string>Si&amp;ze</string>
      </property>
      <addaction name="action16"/>
      <addaction name="action24"/>
      <addaction name="action32"/>
     </widget>
     <addaction name="actionMovable"/>
     <addaction name="separator"/>
     <addaction name="menuStyle"/>
     <addaction name="separator"/>
     <addaction name="menuSize"/>
     <addaction name="separator"/>
     <addaction name="actionDefault"/>
    </widget>
    <addaction name="menuToolbars"/>
    <addaction name="actionStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionAlwaysOnTop"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>F&amp;ilter</string>
    </property>
    <addaction name="actionInfomation"/>
    <addaction name="actionWarning"/>
    <addaction name="actionCritical"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>&amp;File</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarFilter">
   <property name="windowTitle">
    <string>F&amp;ilter</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarHelp">
   <property name="windowTitle">
    <string>&amp;Help</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/o_cancel.png</normaloff>:/images/o_cancel.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_save.png</normaloff>:/images/16_save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_about.png</normaloff>:/images/16_about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_help.png</normaloff>:/images/16_help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionMovable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Movable</string>
   </property>
  </action>
  <action name="actionIconOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Icon only</string>
   </property>
  </action>
  <action name="actionTextOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Text only</string>
   </property>
  </action>
  <action name="actionTextBesideIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text &amp;beside icon</string>
   </property>
  </action>
  <action name="actionTextUnderIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text &amp;under icon</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status &amp;bar</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Always on top</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>&amp;Default</string>
   </property>
  </action>
  <action name="actionInfomation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_information.png</normaloff>:/images/16_information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Infomation</string>
   </property>
  </action>
  <action name="actionWarning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_warning.png</normaloff>:/images/16_warning.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Warning</string>
   </property>
  </action>
  <action name="actionCritical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16_critical.png</normaloff>:/images/16_critical.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Critical</string>
   </property>
  </action>
  <action name="action16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;16x16</string>
   </property>
  </action>
  <action name="action24">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;24x24</string>
   </property>
  </action>
  <action name="action32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;32x32</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
